### DISCUSSION TITLE
My Handy Tool Exploration Journey - Tw93

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#178] [General]
# My Handy Tool Exploration Journey - Tw93

Hi, I'm Tw93, a Frontend Engineer from China, also dabbling a bit in Swift, Rust, Python, and Sketch. I'm the creator of MiaoYan, Pake, Trendy Weekly, XRender, and WeexUi. Believing in long-termism, I am passionate about open source and enjoy experimenting with new ideas.ShareRecently, taking advantage of the holiday downtime, I organized some of the better Mac software, websites, and lifestyle goods I've used over the year. Adhering to the principles of simplicity, usability, and aesthetics, I hope this can offer some assistance when you find yourself puzzled with choices.

https://tw93.fun/en/2024-02-09/tools.html

**Comment by Svec-Tomas** [comment]
I have really enjoyed reading this article, thanks for publishing this on Reddit! My RSS reader catched it before it got deleted and I believe it was never published on Reddit. I love my RSS Reader - News Explorer is my favorite RSS reader and you may love it as well. There is a 14-day free trial if you download it from their website: https://betamagic.nl/products/newsexplorer.html

I was really looking forward to try out Top Calendar, but clicking your link brings me to App Store, where it says, not available in my region. AppRaven says it was removed from the App Store on 10. February, maybe that's the reason? :( I am not sure if this is accurate, but if so, it is really sad as it looked really great!

MessAuto is the next one, that catched my eye. I am using a MacBook pro 14-inch M1 Pro, but when I try to install the MessAuto I get error that it is damaged can't be opened and I should move it to bin. Very strange.

Instead of Bob I am using EasyDict which seems to be working better for me and is also better localised to English.

Instead of Dozer, which seems to be abandoned I have just recently discovered Ice - https://github.com/jordanbaird/Ice - which is in active development with frequent releases.

How is TotalFinder working for you? It looks like it was also abandoned and not optimised for Sonoma. I have read some complaints from other users not working good.

**Reply by tw93** [reply]
Thanks for the feedback, of which I prefer Bob, because EasyDict actually mimics it, TotalFinder can be used, but you need to refer to his official documentation to configure the system is fine!
